build.sbt

```
@@ -92,6 +92,18 @@ lazy val slf4j = project
     }
   )
 
+lazy val `js-console` = project
+  .settings(commonSettings)
+  .dependsOn(core.js)
+  .settings(
+    name := "log4cats-js-console",
+    tlVersionIntroduced := List("2.12", "2.13", "3").map(_ -> "2.6.0").toMap,
+    libraryDependencies ++= Seq(
+      "org.typelevel" %%% "cats-effect" % catsEffectV
```

Is kernel sufficient?

```suggestion
      "org.typelevel" %%% "cats-effect-kernel" % catsEffectV
```

---

Addressed in 3448fec3.